packages/bot-web-ui/src/components/journal.jsx

```diff
@@ -1,63 +1,49 @@
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
+import ContentLoader from 'react-content-loader';
 import { Checkbox, Icon, ThemedScrollbars } from '@deriv/components';
 import { localize } from '@deriv/translations';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
 import { message_types } from '@deriv/bot-skeleton';
 import { connect } from '../stores/connect';
 import '../assets/sass/journal.scss';
+import { contract_stages } from '../constants/contract-stage';
 
 const DateItem = ({ date, time }) => {
     return (
-        <div>
-            <span className='journal__table--date'>{date}</span>
-            <span className='journal__table--time'>{time}</span>
-        </div>
+        <>
+            <span className='journal__text-date'> {date} </span> | <span className='journal__text-time'> {time} </span>
+        </>
     );
 };
 
 const FormatMessage = ({ message }) => {
-    if (React.isValidElement(message)) {
-        return message;
-    }
-    if (typeof message === 'object') {
-        return JSON.stringify(message);
-    }
-    if (typeof message !== 'string') {
-        return message;
+    if (typeof message !== 'string' || !message.includes(':')) {
+        return <div className='journal__text'>{message}</div>;
     }
 
-    // TODO: Below won't work for localised strings.
-    const key_words = ['Bought', 'Sold', 'Profit amount', 'Loss amount'];
     const messages = message.split(':');
 
-    if (messages.length < 2) {
-        return message;
-    }
-
     const title = messages[0];
     const value = messages.slice(1).join(':');
 
     let title_color, value_color;
 
     switch (title) {
-        case key_words[0]: {
-            // Bought
+        case 'Bought': {
             title_color = 'info';
             break;
         }
-        case key_words[1]: {
-            // Sold
+        case 'Sold': {
             title_color = 'error';
             break;
         }
-        case key_words[2]: {
-            // Profit amount
+        case 'Profit amount': {
             value_color = 'success';
             break;
         }
-        case key_words[3]: {
-            // Loss amount
+        case 'Loss amount': {
```

Comment:
Could we get rid of this completely? The logic is language sensitive which doesn't work in a multilingual website.

Reply:
Make sense. 👍